models/comments.model.js

```diff
@@ -0,0 +1,31 @@
+const db = require("../db/connection");
+
+exports.selectCommentById = ({comment_id}) => {
+  return db
+    .query(`
+    SELECT *
+    FROM comments
+    WHERE comment_id = $1;`, [comment_id])
+
+    .then(({ rows }) => {
+      const comment = rows[0];
+      if (!comment) {
+        return Promise.reject({ status: 404, msg: "not found" });
+      }
+    });
+};
+
+exports.removeCommentById = ({ comment_id }) => {
+  return db
+    .query(
+      `DELETE FROM comments
+   WHERE comment_id = $1
+   RETURNING *;`,
+      [comment_id]
+    )
+    .then(({ rows }) => {
+      if (rows) {
+        return;
+      }
+    });
+};
```

Comment:
alternative way to approach doing this deletion whilst also making sure it's a valid comment, you can use the rowCount that comes from the query's return value - if something successfully deletes, the rowCount has 1, if nothing deletes then it's 0 - you could use this to validate if you should reject with a not found or if all is good without doing two queries